### PR TITLE
Add Django 1.9 and specific allow_failures to travis

### PR DESCRIPTION
**Pull request by Anaethelion**
To ensure we work well with the last version

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,6 +30,10 @@ matrix:
       env: DJANGO_VERSION=1.6 DATABASE=sqlite
 
   allow_failures:
+    - python: 3.5
+      env: DJANGO_VERSION=dev DATABASE=sqlite
+    - python: 3.5
+      env: DJANGO_VERSION=dev DATABASE=postgres
```
Comment (gutard) [inline .travis.yml]: This version of python is not declared in `python:` section

File: .travis.yml [hunk 2]
```diff
@@ -35,6 +42,10 @@ matrix:
       env: DJANGO_VERSION=dev DATABASE=sqlite
     - python: 3.3
       env: DJANGO_VERSION=dev DATABASE=postgres
+    - python: 3.3
+      env: DJANGO_VERSION=1.9 DATABASE=sqlite
+    - python: 3.3
+      env: DJANGO_VERSION=1.9 DATABASE=postgres
```
Comment (gutard) [inline .travis.yml]: Should be in `exclude:` section rather than `allow_failures:` one

File: .travis.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@ python:
   - 3.4
 
 env:
+env:
```
Comment (PetrDlouhy) [inline .travis.yml]: Remove the double line.

**Comment by leplatrem** [comment]
Closing because was handled in #133

**Comment by Anaethelion** [comment]
:+1:

**Pull request closed without merging**